jobs/syslog-configuration-test/templates/post-deploy.sh

```diff
@@ -28,7 +28,7 @@ load_ryslogd_config_check() {
 }
 
 ensure_rsyslog_imfile_module_loaded() {
-  [[ $RSYSLOG_CONFIG_CHECK = *"loading module '/usr/lib/rsyslog/imfile.so'"* ]] ||
+  [[ $RSYSLOG_CONFIG_CHECK = *"imfile.so'"* ]] ||
```

Comment:
Could we still check that the message contains 'loading module'? I'm worried we'd miss for e.g. file not found errors.